src/drive/styles/filelist.styl

```diff
@@ -115,11 +114,10 @@
     padding-left 0
 
 .fil-content-cell.fil-content-file
-    position       relative
+    flex 0 0 35%
```

Comment:
I don't like magic value, 35%, 5%. Why?

Reply:
Every column in the file list gets a % of the total width. The file part was 40% before, now it's 5% for the icon and 35 for the file name. Would you be more comfortable with it the % values had names like `column-width-icon`?

Reply:
Yep having a centralized place with variables should be enough. We can do it latter